elasticsearch-core/src/main/scala/com/sumologic/elasticsearch/restlastic/dsl/QueryDsl.scala

```diff
@@ -151,7 +151,7 @@ trait QueryDsl extends DslCommons with SortDsl {
     }
   }
 
-  case class RangeQuery(key: String, bounds: RangeBound*) extends Query {
+  case class RangeQuery(key: String, bounds: RangeBound*) extends Query with Filter {
```

Comment:
Seems like we can already mix-in this trait in code that uses RangeQuery when instantiating the query. 

Still, I wouldn't mind adding it here by default for convenience.

Reply:
A RangeFilter looks good. Although I will create a RangeFilter instead of extending the RangeQuery to be consistent with the existing code style. Also, a Query (with score) and a Filter (no scoring) do have their differences.